cdk/yarn.lock

```diff
@@ -3038,7 +3038,7 @@ debug@^3.1.0:
   dependencies:
     ms "^2.1.1"
 
-debuglog@*, debuglog@^1.0.1:
```

Comment:
I'm a bit surprised this is changing, I don't think I've touched the `cdk` dependencies in this PR 😕